## scan to map 参数
if_debug: false
save_pcd: false

# 坐标系
odom_frame: "odom"
base_frame: "base_link"
map_frame: "map"
lidar_frame: "base_link"

# 匹配结果方差
Variance_X: 0.2
Variance_Y: 0.2
Variance_Yaw: 0.1

Use_TfTree_Always: true

# ICP定位参数
SCORE_THRESHOLD_MAX: 0.13     # 达到最大迭代次数后，代价仍高于此值，认为无法收敛
ANGLE_SPEED_THRESHOLD: 10     # 角速度阈值，角速度大于此值则不发送定位
ANGLE_THRESHOLD: 0.01         # 最小变换角度
DIST_THRESHOLD: 0.01          # 最小变换距离
ANGLE_UPPER_THRESHOLD_: 10    # 最大角度变换
Point_Quantity_THRESHOLD: 180 # 点云数阈值

Maximum_Iterations: 50        # 最大迭代次数

ObstacleRemoval_Distance_Max: 1.0     # 如果雷达点云中点在地图点云最近点大于此值，就认为该点为障碍点
VoxelGridRemoval_LeafSize: 0.1        # 体素滤波的边长

# 定位禁区  判断为定位禁区得到两个点 [x1, y1, x2, y2]
# location_restricted_zone: [6.5, -4.0, 8.8, -3.5,  #己方8度坡
#                           7.4, 3.0, 9.5, 4.5,     #对面8度坡
#                           2.0, 5.0, 6.0, 8.0,     #己方9度坡
#                           8.5, -8, 13.5, -5.0]    #对面9度坡
location_restricted_zone: []                          



# 重定位参数
Relocation_Weight_Score: 1            # 重定位分数系数
Relocation_Weight_Distance: 1         # 重定位距离系数
Relocation_Weight_Yaw: 1              # 重定位yaw角系数

Relocation_Maximum_Iterations: 80     # 重定位最大迭代次数
Loss_Num_Threshold: -1                # 定位丢失次数阈值
Relocation_Score_Threshold_Max: 0.8   # 重定位代价阈值，低于此值认为正确重定位